README.md

```diff
@@ -69,7 +69,7 @@ To learn the commands that you can use with the rainbow bridge run
 Alternatively, clone this repo, `yarn install`, then you can see what commands you can use with:
 
 ```
-    ./index.js --help
+    cli/index.js --help
```

Comment:
Throughout the README the command `rainbow` is being used. Either change it to use `cli/index.js` everywhere, or use `rainbow` here.

Reply:
The `rainbow --help` command is mentioned a few lines above. 
Therefore, I believe that it's good to have `cli/index.js` here, so if someone decides to test using local code, it's clear.

Reply:
On a second thought, for the sake of consistency, can you remove first two instructions:
```
(1) npm i -g rainbow-bridge-cli
(2) rainbow --help
```

Since we are not releasing new versions. If a developer tries that it will just fail. So let's stick with the `cli/index.js` interface for now on all the steps.